tests/e2e/export.go

```diff
@@ -15,17 +15,57 @@
 package e2e
 
 import (
+	"context"
+	"fmt"
 	"path/filepath"
 	"strings"
 
 	"github.com/GoogleCloudPlatform/k8s-config-connector/config/tests/samples/create"
 	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/cli/cmd/export"
+	"github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct/registry"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/yaml"
 )
 
+func directExport(u *unstructured.Unstructured, c client.Client) ([]byte, error) {
+	ctx := context.TODO()
```

Comment:
Nit: probably easier to accept a `ctx context.Context` parameter on new methods (the caller can always pass context.TODO if needed)